test/remote/check_tests.js

```diff
@@ -28,9 +28,15 @@ test(
 
 var client = restify.createJsonClient({
   url: 'http://127.0.0.1:' + config.listen.port
-});
+})
+
+var ACTIONS = [
+  'accountCreate',
+  'accountLogin',
+  'passwordChange'
+]
 
-['accountCreate', 'accountLogin', 'passwordChange'].forEach(function (action) {
+ACTIONS.forEach(function (action) {
```

Comment:
@kitcambridge You magnificent bastard! You are right! That's why you're a dev and I'm just a QE.
I guess it was masked before because line 31 just happened to have a bonus semi-colon which was glueing this thing together.

Oh ASI... :wagsfinger: